### ISSUE TITLE
[Design] [Content] Create wireframe with content for when Veteran selects direct link after 30 days

### ISSUE THREAD

**Issue by benbrasso-agile6** [department-of-veterans-affairs/va.gov-team#68923]
## Tasks

We will have a scenario where a Veteran selects a direct link (from email, for example) to file a claim for a specific appt/visit and it's past the 30 days of eligibility. Therefore, we need a card component to reflect that type of messaging. E.g, they can't file for this claim any longer because it's past 30 days. 

[Here's the card component used for when the Veteran is within the 30 days](https://www.sketch.com/s/0e890de3-2530-4ee0-986e-cf0314334aec/a/elzkWzP). 

- [ ] Decide on component, e.g., card vs. alert? 
- [ ] Create wireframe with content

**Comment by benbrasso-agile6** [comment]
@loripusey we or Appointments team will need this work done at some point. I wanted to document it somewhere. So, here it is. Would you recommend moving it somewhere else? I think you were keeping some of the more PCI-related tickets in a separate epic. Maybe?

**Comment by loripusey** [comment]
so, you're saying that you won't do any prototype wireframes for this but you didn't want to lose site of the work that will need to get done when we go to implement it?

**Comment by loripusey** [comment]
I can create an Epic for our team's implementation in our product and put it in the Ideas column; I'll move this there after I create it

**Comment by benbrasso-agile6** [comment]
Correct, don't need it for prototype. But it's a scenario I just thought of that would happen, and one of the teams would need to solve/design for as an implementation phase arrives.

**Comment by benbrasso-agile6** [comment]
I'm going to pull this into our current sprint now, but for now, should probably only stick to design... until Kelly has more time next sprint to help

**Comment by benbrasso-agile6** [comment]
We'll have to do it in refinement today even though we pulled it in. Here, why don't I split this ticket out into just design. @ytsaoca can you point this please?

**Comment by benbrasso-agile6** [comment]
I added this page into the Travel Pay Past Appts Userflow as well - https://www.sketch.com/s/0e890de3-2530-4ee0-986e-cf0314334aec/p/55601245-1C43-4A62-9DF8-2C830998B53E/canvas#Version

I had added that last night. I know @ytsaoca is going to design office hours today to task about the alert. But, @kellysmith1008 feel free to double check that page/content. Direct link to artboard - https://www.sketch.com/s/0e890de3-2530-4ee0-986e-cf0314334aec/a/g1J9mPq#Version

**Comment by benbrasso-agile6** [comment]
@ytsaoca didn't want you to forget that I left a comment for you in sketch for this ticket
